### ISSUE TITLE
No class list upload button?

### ISSUE THREAD

**Issue by RPriestley** [al-caughey/Google-Meet-Attendance#85]
I see that I can backup my class lists in the settings area. However, I do not see the ability to upload a previously back-uped list?

**Comment by Eagle-Man** [comment]
No, there is no upload button.  Just copy the text and paste it into the class list edit mode.

**Comment by AlCaughey** [comment]
Yeah. I took that button out to reduce clutter in the toolbar (&, TBH, I
thought that I was the only one who used it)

Oops!

I was thinking about adding a new feature to resurrect class lists from
previously saved html files... but it'll take a while to get started on that
